types.py

```diff
@@ -1,7 +1,10 @@
-from LSP.plugin.core.typing import Any, Callable, List, Literal, TypedDict, TypeVar
+from LSP.plugin.core.typing import Any, Callable, List, Literal, Tuple, TypedDict, TypeVar
 
 T_Callable = TypeVar("T_Callable", bound=Callable[..., Any])
 
+StPoint = int
```

Comment:
This `StPoint` looks like `Saint Point` 😄 
Maybe we should rather call it `SublimeTextPoint`

Reply:
Saint Point, a lovely fellow